server/resolvers/TestPlanVersion/directoryResolver.js

```diff
@@ -0,0 +1,5 @@
+const directoryResolver = testPlanVersion => {
+    return testPlanVersion.metadata.directory || '';
+};
+
+module.exports = directoryResolver;
```

Comment:
We can do this later, but in the future it would be better to store this as a column in the database - I think we agreed on that before. But it can stay like this for now since adding migrations would be a pain with a frontend-focused PR.

Reply:
Had the same thought when doing so! Agreed!